pom.xml

```diff
@@ -159,6 +159,8 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <jersey2.version>2.34</jersey2.version>
 
     <!-- jackson versions -->
+    <jackson1.version>1.9.13</jackson1.version>
+    <jackson-jaxr.version>1.9.13</jackson-jaxr.version>
```

Comment:
Please add a separate version property for `jackson-mapper-asl`.  It tends to have a patch version.

On the other hand, I don't think we need a separate version property for `jackson-jaxrs`, it can use the generic `jackson1-version`.

Reply:
Both jackson-mapper-asl & jackson-core-asl have the same patch version. jackson-jaxrs doesn't seem to have a patched version.